config/routes.rb

```diff
@@ -1,5 +1,8 @@
 MANIACChess::Application.routes.draw do
   devise_for :users
+ resource :board, only: [:show]
```

Comment:
Informational: When you're setting up a resourceful route for a single action, you don't need use the array notation. In other words, this line can just be `resource :board, only: :show`.